oak-search/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,38 @@
+<!--
```

Comment:
Why do we need this separate logback config for ?

Reply:
I believe I need this to use the LogCustomizer for comparing my log output

Reply:
ok